code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/ProjectResourceManager.java

```diff
@@ -84,7 +86,7 @@ public Object listInternal(SchemaFactory schemaFactory, String type, Map<Object,
                     throw new ClientVisibleException(ResponseCodes.NOT_FOUND);
                 }
             }
-            policy.grantObjectAccess(project);
+            giveProjectAccess(project);
```

Comment:
I liked the simplicity of this method signature and since it is the same pointer and doesn't make a new object i went for that way. But I could do that as well.